fork/func.py

```diff
@@ -0,0 +1,7 @@
+def proportional(*args):
+	sum_before = sum(args[:-1])
+	*params, special, sum_after = args
+	coeff = sum_after / sum_before
+	new = list(x * coeff for x in params)
+	last = sum_after - sum(new)
+	return (*new, last)
```

Comment:
we don't need free length
not clear
divide zero
negative
you didn't read in folder fork with your name

Reply:
special?
sumafter is not correct